### ISSUE TITLE
Error: "Table of Contents appears to be caption start.."

### ISSUE THREAD

**Issue by brokentool** [gsautter/goldengate-qualitycontrol#6]
Again new for me. Articles I have previously worked on, now started to show this error, which I have not seen before. The ToC title is recognized as a caption. Perhaps due to the word **table**?

I mark it as FP/DE, but wanted to point it out, as every ToC title I came across today seems to be affected.

**Comment by gsautter** [comment]
You assume correctly ... it's simply the word "Table". In the past, we've had a good few documents with missed captions due to font size variation outside the template configured envelope. This is pretty much a reaction to that, to make sure to find all the captions ("Figure" would also trigger an error if not in a caption).

I'm going to evaluate the protocols at some point and assess which checks cause nothing but false positives, and will remove those. Right now, I'm trying to catch the oddities we've encountered and found more or less by chance.

Bottom line: No worries, "Table of Contents" is simply an FP. Just one that's hard to prevent in this kind of check.
